go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/brancz/kube-rbac-proxy
 go 1.13
 
 require (
+	github.com/campoy/embedmd v1.0.0 // indirect
```

Comment:
not sure why this is added?

Reply:
I run `make` and `go mod tidy` and it was added automatically. Removed it as it is not needed.